pom.xml

```diff
@@ -1181,7 +1181,7 @@
                         <image>
                             <repository>${dockerHubPublic}/library/zookeeper</repository>
                             <tag>3.9.2-jre-17</tag>
-                            <digest>sha256:e9a4fd7748598e7eac08cab3d28d7b2e6f092d5ec6509af3821b100924ceecb4</digest>
+                            <digest>sha256:62bc819222e46b25a83557f501923cb6a7e4c15baf14e767196f5a0cf4c7e555</digest>
```

Comment:
```suggestion
```

Reply:
@kusumaghoshdastidar is 3.9.2-jre-17 not a static tag then build fails due to wrong digest, so I update it, build passes, next build fails due to wrong digest.

Reply:
@kusumaghoshdastidar had not realised we could just remove it